templates/managed-rds-postgresql-v1.j2

```diff
@@ -0,0 +1,284 @@
+        # When there is change in the contents of json file in the metadata section, cfn-hup will call cfn-init.
+        cfn_hup_service:
+          files:
+            /etc/cfn/cfn-hup.conf:
+              content: !Sub |
+                [main]
+                stack=${AWS::StackId}
+                region=${AWS::Region}
+                verbose=true
+                interval=5
+              mode: "000400"
+              owner: root
+              group: root
+            /etc/cfn/hooks.d/cfn-auto-reloader.conf:
+              content: !Sub |
+                [cfn-auto-reloader-hook]
+                triggers=post.update
+                path=Resources.DBInstance.Metadata.AWS::CloudFormation::Init
+                action=/opt/aws/bin/cfn-init -v --stack ${AWS::StackName} --resource DBInstance --configsets SetupCfn,SetupVolume --region ${AWS::Region}
+              mode: "000400"
+              owner: root
+              group: root
+        /lib/systemd/system/cfn-hup.service:
+          content: !Sub |
+            [Unit]
+        Description=cfn-hup daemon
+
+        [Service]
+        Type=simple
+        ExecStart=/opt/aws/bin/cfn-hup
+        Restart=always
+
+        [Install]
+        WantedBy=multi-user.target
+        mode: "000400"
+        owner: root
+        group: root
+    commands:
+      01_enable_cfn-hup:
+        command: "/bin/systemctl enable cfn-hup.service"
+      02_start_cfn-hup:
+        command: "/bin/systemctl start cfn-hup.service"
+        tag_volume:
+      commands:
+        01_tag_root_disk:
+          command: !Join
+            - ''
+            - - "DB_INSTANCE_ID=$(/usr/bin/curl -s http://169.254.169.254/latest/meta-data/instance-id) "
+              - "ROOT_DISK_ID=$(/usr/bin/aws ec2 describe-volumes --region $AWS_REGION "
+          - "--filters Name=attachment.instance-id,Values=$DB_INSTANCE_ID "
+          - "--query Volumes[].VolumeId --out text); "
+          - "/usr/bin/aws ec2 create-tags --region $AWS_REGION --resources $ROOT_DISK_ID "
+          - "--tags Key=Name,Value=$DB_INSTANCE_ID-root Key=cloudformation:stack-name,Value=$STACK_NAME "
+          - "Key=Department,Value=$DEPARTMENT Key=Project,Value=$PROJECT Key=OwnerEmail,Value=$OWNER_EMAIL"
+          env:
+            AWS_REGION: !Ref AWS::Region
+            STACK_NAME: !Ref AWS::StackName
+            DEPARTMENT: !Ref Department
+            PROJECT: !Ref Project
+            OWNER_EMAIL: !Ref OwnerEmail
+    Properties:
+      AllocatedStorage: !Ref 'VolumeSize'
+      DBInstanceClass: !Ref 'InstanceType'
+      KmsKeyId: !If [HasKmsKeyId, !Ref KmsKeyId, !Ref 'AWS::NoValue']
+      StorageEncryption: !Ref 'EncryptVolume'
+      Engine: postgres
+      EngineVersion: !Ref 'DBEngineVersion'
+      MasterUsername: !Ref 'DBMasterUsername'
+      MasterUserPassword: !Ref 'DBMasterUserPassword'
+      MultiAZ: !Ref 'MultiAZ'
+      BackupRetentionPeriod: !Ref 'DBBackupRetentionPeriod'
```

Comment:
I think you need a `DBSubnetGroupName` otherwise it will get put in the default VPC.  You want to put it in the `sandcastlevpc` for org-sagebase-sandbox and `snowflakevpc` in org-sagebase-scicomp account.

Reply:
What if the subnet name is in the highest level template? Like in the provisioner request: https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/303

Reply:
I missed adding this as a parameter

Reply:
I still don't see it?